OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -477,6 +487,18 @@ public static void unsubscribeWhenNotificationsAreDisabled(boolean set) {
       mUnsubscribeWhenNotificationsAreDisabled = set;
    }
 
+   /**
+    * If {@link NotificationWillShowInForegroundHandler} is not set, the ability to control a
+    *    default {@link OSNotificationDisplay} should still exist
+    */
+   public static void setNotificationDisplayOption(OSNotificationDisplay displayOption) {
```

Comment:
Redundant APIs create another surface which has a lot of dependencies:
1. More SDK code and code paths
2. Unit Tests
3. Manual testing
4. Documentation & Examples

Not saying we would never want a redundant API, just it should have a good use case. In this case if they wanted to not show a notification when the app is in focus for some reason I can't see way they won't want to setup a notifiationWillShowInForegorund handler.